apps/discord/discord.talon

```diff
@@ -18,7 +34,9 @@ current call: user.discord_go_current_call()
 toggle pins: user.discord_toggle_pins()
 toggle inbox: user.discord_toggle_inbox()
 toggle (members | member list): user.discord_toggle_members()
+toggle (vee see text | voice text): user.discord_toggle_members()
```

Comment:
Have you tried this with `v c text` instead? This usually works fine with conformer and is a bit more intention revealing.